app/components/publish-actions.tsx

```diff
@@ -44,21 +45,37 @@ type TriggeredPopoverProps = {
     setAnchorEl: (el: HTMLElement | undefined) => void
   ) => React.ReactNode;
   popoverProps: Omit<PopoverProps, "open" | "anchorEl" | "onClose">;
+  trigger?: HTMLElement;
 };
 
 export const TriggeredPopover = (props: TriggeredPopoverProps) => {
-  const { children, renderTrigger, popoverProps } = props;
+  const { children, renderTrigger, popoverProps, trigger } = props;
   const [anchorEl, setAnchorEl] = useState<Element | undefined>();
+
+  useEffect(() => {
+    setAnchorEl(trigger);
+  }, [trigger]);
```

Comment:
Nit: I think you can achieve the same behavior with an initializer function like:

```typescript
const [anchorEl, setAnchorEl] = useState<Element | undefined>(() => trigger);
```

or maybe even

```typescript
const [anchorEl, setAnchorEl] = useState<Element | undefined>(trigger);
```
?

Reply:
@bprusinowski Not quite sure why but I tested both before I created this PR and it didn't work